requirements.txt

```diff
@@ -1,6 +1,5 @@
 alembic==1.7.7
 click==8.0.4
-dataclasses==0.8
```

Comment:
We should be good without these so I'm going to go ahead and omit them from now. Moving forward I want to try to refrain from removing anything from the requirements.txt file